src/main/io/serial_msp.h

```diff
@@ -20,11 +20,258 @@
+#define MSP_SET_RAW_GPS          201    //in message          fix, numsat, lat, lon, alt, speed
+#define MSP_SET_PID              202    //in message          P I D coeff (9 are used currently)
+#define MSP_SET_BOX              203    //in message          BOX setup (number is dependant of your setup)
+#define MSP_SET_RC_TUNING        204    //in message          rc rate, rc expo, rollpitch rate, yaw rate, dyn throttle PID, yaw expo
+#define MSP_ACC_CALIBRATION      205    //in message          no param
+#define MSP_MAG_CALIBRATION      206    //in message          no param
+#define MSP_SET_MISC             207    //in message          powermeter trig + 8 free for future use
+#define MSP_RESET_CONF           208    //in message          no param
+#define MSP_SET_WP               209    //in message          sets a given WP (WP#,lat, lon, alt, flags)
+#define MSP_SELECT_SETTING       210    //in message          Select Setting Number (0-2)
+#define MSP_SET_HEAD             211    //in message          define a new heading hold direction
+#define MSP_SET_SERVO_CONFIGURATION 212    //in message          Servo settings
+#define MSP_SET_MOTOR            214    //in message          PropBalance function
+#define MSP_SET_NAV_CONFIG       215    //in message          Sets nav config parameters - write to the eeprom
+#define MSP_SET_3D               217    //in message          Settings needed for reversible ESCs
+#define MSP_SET_RC_DEADBAND      218    //in message          deadbands for yaw alt pitch roll
+#define MSP_SET_RESET_CURR_PID   219    //in message          resetting the current pid profile to defaults
+#define MSP_SET_SENSOR_ALIGNMENT 220    //in message          set the orientation of the acc,gyro,mag
+
+// #define MSP_BIND                 240    //in message          no param
+
+#define MSP_EEPROM_WRITE         250    //in message          no param
+
+#define MSP_DEBUGMSG             253    //out message         debug string buffer
+#define MSP_DEBUG                254    //out message         debug1,debug2,debug3,debug4
+
+// Additional commands that are not compatible with MultiWii
+#define MSP_UID                  160    //out message         Unique device ID
+#define MSP_ACC_TRIM             240    //out message         get acc angle trim values
+#define MSP_SET_ACC_TRIM         239    //in message          set acc angle trim values
+#define MSP_GPSSVINFO            164    //out message         get Signal Strength (only U-Blox)
+#define MSP_SERVO_MIX_RULES      241    //out message         Returns servo mixer configuration
+#define MSP_SET_SERVO_MIX_RULE   242    //in message          Sets servo mixer configuration
+#define MSP_SET_1WIRE            243    //in message          Sets 1Wire paththrough
+
 // Each MSP port requires state and a receive buffer, revisit this default if someone needs more than 2 MSP ports.
 #define MAX_MSP_PORT_COUNT 2
 
-void mspInit(serialConfig_t *serialConfig);
+typedef enum {
```

Comment:
i see two solutions: 1) new header file as @ledvinap suggested, 2) re-arrange contents and seperate public from from private implementation details and add a comment :smile: 

I think we'll merge as is, since there is much value in the tests now, and I will accept a PR to cleanup once concensus is reached.